app/assets/stylesheets/styles.css

```diff
@@ -2596,6 +2602,13 @@ textarea.form-control-lg {
   height: calc(3.5rem + 2px);
   padding: 1rem 0.75rem;
 }
+
+.form-floating > .form-control-error,
+.form-floating > .form-select {
+    height: calc(3.5rem + 2px);
```

Comment:
I see that this same calc is being used in other places, does it make sense to move it to a variable? also, what it's supposed to accomplish?

Reply:
this file could use a full refactor, will ignore for now